### DISCUSSION TITLE
en/article/2022/10/20/chunked-git-push-into-azure-devops/

### DISCUSSION THREAD

**Discussion by giscus** [michaelcontento/michaelcontento.github.io#20] [General]
# en/article/2022/10/20/chunked-git-push-into-azure-devops/

Blog of a passionated full-stack developer around C#, Github and Azure

https://www.michaelcontento.de/en/article/2022/10/20/chunked-git-push-into-azure-devops/

**Comment by harrybin** [comment]
Hi,
have you tried scalar? (https://github.com/microsoft/scalar)
Maybe scalar solves this problem too?
But I guess the main point is the initial commit, right?
So scalar may be a good tool later on, when working with such a big repo.

Greetings
Harry
